lib/manager/pip_requirements/package.js

```diff
@@ -40,6 +40,8 @@ async function getPackageUpdates(config) {
         depName,
         newVersion,
         newVersionMajor: getMajor(newVersion),
+        changeLogFromVersion: currentVersion,
+        changeLogToVersion: newVersion,
```

Comment:
@rarkins I had to add this for things to work, any case where `currentVersion != changeLogFromVersion`?

Reply:
Yes, the case is when ranges are in use, e.g. going from `^1.0.0` to `^2.0.0`